pest/examples/parens.rs

```diff
@@ -67,9 +67,20 @@ fn main() {
         io::stdin().read_line(&mut line).unwrap();
         line.pop();
 
-        match ParenParser::parse(Rule::expr, &line) {
+        let parsed = ParenParser::parse(Rule::expr, &line);
+        #[cfg(feature = "miette")]
+        let parsed = parsed
+            .map_err(Error::into_miette)
+            .map_err(miette::Report::from);
+
+        match parsed {
             Ok(pairs) => println!("{:?}", expr(pairs)),
-            Err(e) => println!("\n{}", e),
+            // To print pest errors, use Display formatting.
+            #[cfg(not(feature = "miette"))]
+            Err(e) => eprintln!("\n{}", e),
+            // To print miette errors, use Debug formatting.
+            #[cfg(feature = "miette")]
+            Err(e) => eprintln!("\n{:?}", e),
```

Comment:
Is this true? (Does `impl Debug for Error` use the miette formatting? If not yet, doing so (when alt is false) seems like a good idea.)

Reply:
`impl Debug for Error` does not, but `impl Debug for Report` does. And in this scenario, `e: Report`. I honestly don't thing that `Debug` for `Error` should print miette-styled messages, it should still be opt-in.

Reply:
I missed the conversion of the error into a `miette::Report` above.